superset/db_engine_specs/hive.py

```diff
@@ -163,10 +163,11 @@ def convert_to_hive_type(col_type: str) -> str:
             bucket_path,
             os.path.join(upload_prefix, table.table, os.path.basename(filename)),
         )
-        # TODO(bkyryliuk): support other delimiters
+
         sql = f"""CREATE TABLE {str(table)} ( {schema_definition} )
-            ROW FORMAT DELIMITED FIELDS TERMINATED BY ',' STORED AS
-            TEXTFILE LOCATION '{location}'
+            ROW FORMAT SERDE 'org.apache.hadoop.hive.contrib.serde2.MultiDelimitSerDe'
+            WITH SERDEPROPERTIES ("field.delim"="{csv_to_df_kwargs['sep']}")
```

Comment:
Yes, this code is really insecure, but aren't we all? Hahahahahahahaha

Reply:
good point, I would suggest to use params for this one: https://docs.sqlalchemy.org/en/13/core/connections.html#sqlalchemy.engine.Engine.execute